enigma/src/main/java/cuchaz/enigma/source/cfr/EnigmaDumper.java

```diff
@@ -273,7 +273,8 @@ public Dumper methodName(String name, MethodPrototype method, boolean special, b
 		int now = sb.length();
 		Token token = new Token(now - name.length(), now, name);
 
-		if (entry != null) {
+		// Skip constructor references
+		if (entry != null && !name.equals("new")) {
```

Comment:
Is this change related, its not clear whats its doing?

Reply:
It's intended, see [here](https://github.com/QuiltMC/enigma/issues/10)